src/main/java/codechicken/nei/ItemPanel.java

```diff
@@ -1,85 +1,83 @@
 package codechicken.nei;
 
-import codechicken.lib.vec.Rectangle4i;
 import codechicken.nei.api.GuiInfo;
 import codechicken.nei.api.INEIGuiHandler;
 import codechicken.nei.guihook.GuiContainerManager;
+import codechicken.nei.ItemsGrid.ItemPanelSlot;
 import codechicken.nei.recipe.GuiCraftingRecipe;
 import codechicken.nei.recipe.GuiRecipe;
 import codechicken.nei.recipe.GuiUsageRecipe;
+import codechicken.nei.recipe.StackInfo;
 import net.minecraft.client.gui.inventory.GuiContainer;
+import net.minecraft.client.gui.inventory.GuiContainerCreative;
 import net.minecraft.inventory.Slot;
 import net.minecraft.item.ItemStack;
+import net.minecraftforge.fluids.FluidStack;
+import net.minecraftforge.fluids.IFluidContainerItem;
 
 import java.util.ArrayList;
 
-import static codechicken.lib.gui.GuiDraw.drawRect;
-import static codechicken.nei.NEIClientConfig.canPerformAction;
-
-public class ItemPanel extends Widget {
-    /**
-     *  Backwards compat :-/
-     */
-    public static ArrayList<ItemStack> items = new ArrayList<>();
-
-    /**
-     * Should not be externally modified, use updateItemList
-     */
-    public ArrayList<ItemStack> realItems = new ArrayList<>();
```

Comment:
same as above

Reply:
This is you variable (see previous block). Need save it?

Reply:
if it boots in GTNH without it, it's probably fine.  Although it'd be safer just to leave it in with @Deprecated